composer.json

```diff
@@ -19,6 +19,8 @@
         "symplify/vendor-patches": "^11.5"
     },
     "require-dev": {
+        "ext-sqlite3": "*",
+        "ext-pdo_sqlite": "*",
```

Comment:
we don't use pdo-sqlite afaik

Reply:
when install sqlite3 extension, it seems install pdo_sqlite as well, if it need to be here, it may need to be moved under `"require"`